2001-3000/LIT2842RepCh125.xml

```diff
@@ -73,15 +78,72 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+               <bibl><ptr target="bm:ZekraAbaw"/></bibl>
+            </listBibl>
+            <listBibl type="secondary">
+               <bibl><ptr target="bm:Grebaut1934Priere"></ptr></bibl>
             </listBibl>
          </div>
-         <div type="edition">
-            <div type="textpart" subtype="incipit" xml:lang="gez">
-               <ab>
-                  ሰላም፡ ለዝክረ፡ ስምከ፡ በመጽሔተ፡ መስቀል፡ ዘተለክዐ።
-               </ab>
+         <div type="edition" xml:lang="gez">
+            <div type="textpart" xml:id="malke">
+               <lg type="stanza" n="1">
+                  <l n="1">ሰላም፡ ለዝክረ፡ ስምከ፡ በመጽሔተ፡ መስቀል፡ ዘተለክአ፨</l>
+                  <l n="2">ወለሥዕርትከ፡ ጸሊም፡ ዘደመ፡ ተኰርኦ፡ ተቀብአ፨</l>
+                  <l n="3">ሰዋስወ፡ ፍቅር፡ ክርስቶስ፡ ትኩል፡ ዲበ፡ እንግድአ፨</l>
+                  <l n="4">አስትየኒ፡ ለተሳትፎ፡ እንተ፡ ሰተይከ፡ ጽዋዐ፨</l>
+                  <l n="5">በዋዕየ፡ ፍቅርከ፡ አምላካዊ፡ ሕሊናየ፡ ጸምአ፨</l>
+               </lg>
+               <lg type="stanza" n="2">
+                  <l n="1">ሰላም፡ ለርእስከ፡ ሥርግወ፡ አክሊል፡ ዘሥዖክ፨</l>
+                  <l n="2">ወለዕጉሠ፡ ምራቅ፡ ገጽከ፡ እምሐራ፡ ጲላጦስ፡ መልአክ፨</l>
+                  <l n="3">ኢየሱስ፡ ክርስቶስ፡ ተወካፌ፡ ስቅለት፡ አምላክ፨</l>
+                  <l n="4">እንበለ፡ ደምከ፡ ወመስቀልከ፡ ቡሩክ፨</l>
+                  <l n="5">አልብየ፡ ምክሕ፡ እምዘመድ፡ ወዐርክ፨</l>
+               </lg>
+               <lg type="stanza" n="26">
+                  <l n="1">ሰላም፡ ለትንሣኤከ፡ እንበለ፡ መዋጥሕ፡ ወግልባቤ፨</l>
+                  <l n="2">ወለዕርገትከ፡ በኀይል፡ ውስተ፡ ደብረ፡ ስኂን፡ ወደብረ፡ ከርቤ፨</l>
+                  <l n="3">በኵረ፡ ትንሣኤ፡ ክርስቶስ፡ ልሳነ፡ መጽሐፍ፡ በከመ፡ ይቤ፨</l>
+                  <l n="4">አጥረይኩ፡ ምክሐ፡ ወጸወነ፡ በጊዜ፡ ምንዳቤ፨</l>
+                  <l n="5">ኪያከ፡ አበ፡ ወኪያከ፡ መጋቤ፨</l>
+               </lg>
+               <lg type="stanza" n="27">
+                  <l n="1">ሰላም፡ ለምጽአትከ፡ በስብሐተ፡ አብ፡ ወላዲ፨</l>
+                  <l n="2">ወለመንግሥትከ፡ ህልው፡ ለዓለመ፡ ዓለም፡ ወዓዲ፨</l>
+                  <l n="3">ጥዑመ፡ ስም፡ ክርስቶስ፡ ሞገሰ፡ ዮሐንስ፡ ቃለ፡ ዐዋዲ፨</l>
+                  <l n="4">እንዘ፡ ትዜከር፡ እግዚእየ፡ ትእይርተ፡ ሄሮድስ፡ ከሐዲ፨</l>
+                  <l n="5">ለቤዛ፡ ኃጥእ፡ ገብርከ፡ መስቀለከ፡ ፍዲ፨</l>
+               </lg>
+            </div>
+            <div type="textpart" xml:id="tarafa-malke">
+               <lg type="stanza" n="1">
+                  <l n="1">ሰላም፡ ለከ፡ ሠናየ፡ መዐዛ፡ ወጽጌ፡ ንጉሠ፡ ዕፀዋት፡ ዕፅ፨</l>
+                  <l n="2">ወመንበረ፡ ስብሐት፡ ግሩም፡ ዘኪሩቤል፡ ቢጽ፨</l>
+                  <l n="3">መስቀል፡ ርቡዐ፡ ገጽ፨</l>
+               </lg>
+               <lg type="stanza" n="2">
+                  <l n="1">ይትባረክ፡ እግዚአብሔር፡ በአፈ፡ ኵሉ፡ ዘሥጋ፨</l>
+                  <l n="2">ወይትአኰት፡ ስሙ፡ ቅድመ፡ ማኅበር፡ ወእንግልጋ፨</l>
+                  <l n="3">ክርስቶስ፡ ስቁል፡ ለአርዌ፡ ብርት፡ ዘትዛወጋ፨</l>
+                  <l n="4">ኅብስተ፡ ሥጋከ፡ ምስለ፡ ማየ፡ ሕይወት፡ ፈለጋ፨</l>
+                  <l n="5">እንበለ፡ ሤጥ፡ ለሰብእ፡ ዘወሀብከ፡ ጸጋ፨</l>
+               </lg>
+               <lg type="stanza" n="3">
+                  <l n="1">ስብሐት፡ ለከ፡ አምላኪየ፡ በኍልቈ፡ ሰብእ፡ ወመላእክት፨</l>
+                  <l n="2">ስብሐት፡ ለከ፡ አምላኪየ፡ በኍልቈ፡ ጽዱላን፡ ከዋክብት፨</l>
+                  <l n="3">ስብሐት፡ ለከ፡ አምላኪየ፡ በኍልቈ፡ | ኵሉ፡ ዓለማት፨</l>
```

Comment:
What is the meaning and relevance of that sign | ?

Reply:
I simply copypaste the xml as produced by @smaugustine .